extensions/catppuccin/src/utils/palette.util.ts

```diff
@@ -0,0 +1,13 @@
+import type { CatppuccinFlavor } from "@catppuccin/palette";
+import { flavors } from "@catppuccin/palette";
+
+export type CatppuccinFlavors = typeof flavors;
+export type FlavorName = keyof CatppuccinFlavors;
```

Comment:
Both of these are already types provided by `@catppuccin/palette` iirc.

Reply:
now using FlavorName from `@catppuccin/palette`